package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.1.0 MVP",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 5173",
```

Comment:
Local için kullanılan default port değişikliği unutulmuş sanırım , kendi localinizde farklı portlarda çalıştırmak için "npm run dev -- -p 5173" & "yarn dev -p 5173" şeklinde kullanabilirsiniz.

Reply:
institution projesi ile aynı scriptlerin kullanılması için bilinçli yapılan bir değişiklikti, ama değiştirilebilir benim için iki projede aynı scripti kullandıktan sonra çok önemli değil.

Reply:
Çok önemli bir konu olduğunu düşünmüyorum ama ben default kalması taraftarıyım , kararı sana bırakıyorum, resolve olarak işaretledim.